<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RisoPrompt</class>
 <widget class="QMainWindow" name="RisoPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Terminal Grotesque</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RisoPrompt</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: rgba(255, 72, 176, 210);
	font-family: &quot;Terminal Grotesque&quot;;
}

QPushButton {
	color: white;
	border-width: 2px;
	border-style: solid;
	border-color: white;
}

QListWidget::item {
	color: rgb(255, 232, 0);
	font-size: 30px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="saveButton">
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>18</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Save the conversation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="loadButton">
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Load a saved conversation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>load</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="newButton">
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Start a new conversation (current coversation will be cleared)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>new</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="modelButton">
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Change the Gemini model in use (current coversation will be cleared)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>model</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="copyButton">
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Copy the conversation to the clipboard&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>copy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="ConversationWidget" name="conversationWidget" native="true">
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Conversation area&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="CustomPlainText" name="promptInput">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>74</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>74</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Terminal Grotesque</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Message input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Loading...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPlainText</class>
   <extends>QPlainTextEdit</extends>
   <header>customplaintext.h</header>
  </customwidget>
  <customwidget>
   <class>ConversationWidget</class>
   <extends>QWidget</extends>
   <header>conversationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
